package.json

```diff
@@ -5,7 +5,7 @@
     "scripts": {
         "start": "npm run clean && npm run serve",
         "serve": "eleventy --serve",
-        "build": "rimraf dist && eleventy",
+        "build": "npm run clean && ELEVENTY_ENV=production eleventy",
```

Comment:
This compound action is very likely not portable. It appears there is an npm package which can have this effect across platforms - https://www.npmjs.com/package/cross-env